package.json

```diff
@@ -39,19 +39,21 @@
     "linter"
   ],
   "dependencies": {
-    "commander": "2.9.0",
-    "gherkin": "4.0.0",
-    "glob": "7.0.5",
-    "lodash": "4.13.1"
+    "commander": "^2.17.1",
+    "gherkin": "^5.1.0",
+    "glob": "^7.1.2",
+    "lodash": "^4.17.10"
   },
   "devDependencies": {
     "babel-cli": "6.26.0",
-    "babel-preset-env": " 1.6.1",
-    "chai": "4.1.1",
-    "eslint": "2.13.1",
-    "mocha": "2.5.3",
-    "mocha-sinon": "1.1.5",
-    "sinon": "1.17.4"
+    "babel-preset-env": "^1.7.0",
+    "chai": "^4.1.2",
+    "coveralls": "^3.0.2",
+    "eslint": "^5.3.0",
+    "mocha": "^5.2.0",
+    "mocha-sinon": "^2.1.0",
+    "nyc": "^12.0.2",
```

Comment:
I am not familiar with nyc. What value is it adding?

Reply:
This package will be moved to the next PR. Is the package used to calculate the coverage ([Istanbul](https://istanbul.js.org/))